src/main/java/zd/zero/waifu/motivator/plugin/assets/AssetDefinitionService.kt

```diff
@@ -0,0 +1,32 @@
+package zd.zero.waifu.motivator.plugin.assets
+
+object AssetDefinitionService {
+
+    fun getRandomCelebrationVisualAsset(): VisualMotivationAssetDefinition {
+        val celebrationAsset = VisualAssetManager.supplyAssetDefinitions()
+            .filter {
+                it.categories.contains(WaifuAssetCategory.CELEBRATION)
+            }.random()
+        return VisualAssetManager.resolveAsset(celebrationAsset)
+    }
+
+    fun getRandomCelebrationAudibleAsset(): AudibleMotivationAssetDefinition {
+        // todo: use remote/local assets
```

Comment:
I figured that we'll eventually migrate the audio/text stuff out of the plugin, so I just hard-coded some stuff for right now.

Reply:
We'll eventually do to only publish the plugin without the bundled assets so we can update the assets dynamically, in time we could also celebrate waifu birthdays! 🎉 so we'll show it to the waifu of the day. For the todo, let's create an issue for it and reference this file so we can pick up todos easily without affecting the codebase.

Reply:
Do you want an issue create and the todo removed out of the code?

Reply:
Yes please, so we can separate code tracking to issue tracking.

Reply:
Done!